src/linter/xmlTemplate/Parser.ts

```diff
@@ -351,11 +352,13 @@ export default class Parser {
 		} else if (namespace === SVG_NAMESPACE) {
 			// Ignore SVG nodes
 			this.#context.addLintingMessage(this.#resourcePath,
-				MESSAGE.SVG_IN_XML,
-				undefined as never,
 				{
-					line: tag.openStart.line + 1, // Add one to align with IDEs
-					column: tag.openStart.character + 1,
+					id: MESSAGE.SVG_IN_XML,
+					args: undefined as never,
```

Comment:
`args` can now be omitted here, right?

Reply:
I have cherry-picked and moved this into another PR: https://github.com/UI5/linter/pull/827
It's fixed there